### ISSUE TITLE
Migration to batou 2 or just migration documentation is broken

### ISSUE THREAD

**Issue by icemac** [flyingcircusio/batou#162]
First step at https://batou.readthedocs.io/en/latest/upgrading.html breaks with:

```
$ curl -sL https://raw.githubusercontent.com/flyingcircusio/batou/master/bootstrap | sh
Running unclean installation from requirements.txt
Creating venv ...
Ensuring pip ...
Ensuring unclean install ...
usage: batou [-h] [-d] {deploy,secrets} ...
batou: error: invalid choice: 'appenv-update-lockfile' (choose from 'deploy', 'secrets')
```

**Comment by icemac** [comment]
#151 seems to fix this problem

**Comment by ctheune** [comment]
Fixed in #167
